brainwidemap/intro_figure_example_time_series.py

```diff
@@ -0,0 +1,334 @@
+from one.api import ONE
+import brainbox.behavior.wheel as wh
+from brainbox.processing import bincount2D
+from ibllib.atlas import AllenAtlas
+from ibllib.atlas.regions import BrainRegions
+from brainbox.io.one import SpikeSortingLoader
+from brainbox.io.one import SessionLoader
+import ibllib
+
+import numpy as np
+from pathlib import Path
+import math
+import matplotlib.pyplot as plt
+import pandas as pd
+import matplotlib as mpl
+
+
+
+mpl.rcParams.update({'font.size': 10})
+
+one = ONE(base_url='https://openalyx.internationalbrainlab.org',
+          password='international', silent=True)
+
+# save results for plotting here
+pth_res = Path(one.cache_dir, 'brain_wide_map', 'motor_correlates')
```

Comment:
Let's not write to the ONE cache with intermediate results, you can either have the user set an input or fall back to creating a directory in home